lib/tasks/order_consumer.rake

```diff
@@ -0,0 +1,34 @@
+namespace :queue do
+
+  RABBIT_URL="amqp://guest:guest@rabbitmq"
+
+  desc "start consuming rabbitMQ messages"
+  task consume: :environment do
+    conn = Bunny.new(RABBIT_URL)
+    conn.start
+    channel = conn.create_channel
+    queue = channel.queue( 'delivery.status_updated', auto_delete: true)
+
+    puts "subscribing"
+    queue.subscribe(block: true) do |_, _, payload|
+  # DO STUFF
+      puts "Consuming!! : " + payload.to_s
+
+      puts "sending to sidekiq worker : " + payload.to_s
```

Comment:
why do you print payload.to_s twice?

Reply:
Thought i deleted one of them guess not :)